packages/hardhat/.prettierrc.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  ...require('../../.prettierrc.json'),
+}
```

Comment:
Looking at the [prettier docs](https://prettier.io/docs/en/configuration.html) I think you can remove this file, prettier will automatically search in parent dirs.

(However, we'd need to reintroduce this if we wanna override/add configuration...)

Reply:
you are right. removed it in [a582afe](https://github.com/yieldgate/yieldgate/pull/29/commits/a582afe1400d71e4338fe0fcace8ab8ee72b9333)